flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -54,6 +54,13 @@ function setup_kubernetes_for_linux {
           chmod +x minikube && sudo mv minikube /usr/bin/minikube
     fi
 
+    # Install the latest version, avoid none driver of minikube
```

Comment:
hm, it looks like you misunderstood me here. This change doesn't add any value because we're already installing minikube in the if block above. My question was whether it would be enough to allow for the `MINIKUBE_VERSION` variable to be set to `latest` if we really want to allow the latest version instead of hard-coding the Minikube version in the variable.

Allowing "latest" was what @wangyang0918 suggested in [the comment](https://github.com/apache/flink/pull/21837#pullrequestreview-1282199706) I referred to [above](https://github.com/apache/flink/pull/23494#pullrequestreview-1684722967). 

But apparently there are some restrictions on using "latest" right-away if I understand @wangyang0918 correctly. I don't understand the restrictions, yet.